zerver/lib/test_classes.py

```diff
@@ -2600,3 +2601,13 @@ def get_generic_payload(self, method: str = "register") -> dict[str, Any]:
         token_kind = PushDeviceToken.FCM
 
         return {"user_id": user_id, "token": token, "token_kind": token_kind}
+
+
+class ThumbnailTestCase(ZulipTestCase):
+    @contextmanager
+    def thumbnail_formats(self, *thumbnail_formats: ThumbnailFormat) -> Iterator[None]:
+        with (
+            mock.patch("zerver.lib.thumbnail.THUMBNAIL_OUTPUT_FORMATS", thumbnail_formats),
+            mock.patch("zerver.views.upload.THUMBNAIL_OUTPUT_FORMATS", thumbnail_formats),
+        ):
+            yield
```

Comment:
If this is the only thing it adds, this helped could just be a `ZulipTestCase` feature, since it doesn't do anything unless accessed.